hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HAUtils.java

```diff
@@ -387,7 +387,16 @@ public static File getMetaDir(DBDefinition definition,
               waitForCACerts(certClient::updateCAList, expectedCount),
               waitDuration);
         } else {
-          caCertPemList = certClient.listCA();
+          List<String> x509Certificates = new ArrayList<>();
```

Comment:
Why define a separate list instead of using `caCertPemList`, like other `if` branches?

Reply:
Did little refactor, have a look at latest change